templates/data/kafka.html

```diff
@@ -18,314 +16,393 @@
+                height="676",
+                hi_def=True,
+                loading="lazy",
+                attrs={"class": "p-image-container__image"}) | safe
+        }}
       </div>
-      <div class="col">
-        <h2 class="p-text--small-caps">
-          Run Apache Kafka<sup>&reg;</sup> on
-        </h2>
-        <div class="p-logo-section">
-          <div class="p-logo-section__items">
-            <div class="p-logo-section__item">
-              {{ image(url="https://assets.ubuntu.com/v1/36a0cd3b-microsoft-azure-2021.png",
-                            alt="Microsoft Azure",
-                            width="78",
-                            height="256",
-                            hi_def=True,
-                            loading="lazy",
-                            attrs={"class": "p-logo-section__logo"},) | safe
-              }}
-            </div>
-            <div class="p-logo-section__item">
-              {{ image(url="https://assets.ubuntu.com/v1/4648a4a4-openstack-logo.png",
-                            alt="OpenStack",
-                            width="509",
-                            height="384",
-                            hi_def=True,
-                            loading="lazy",
-                            attrs={"class": "p-logo-section__logo"},) | safe
-              }}
-            </div>
-            <div class="p-logo-section__item">
-              {{ image(url="https://assets.ubuntu.com/v1/497cabc1-google-cloud-logo.png",
-                            alt="Google cloud",
-                            width="397",
-                            height="256",
-                            hi_def=True,
-                            loading="lazy",
-                            attrs={"class": "p-logo-section__logo"},) | safe
-              }}
-            </div>
-            <div class="p-logo-section__item">
-              {{ image(url="https://assets.ubuntu.com/v1/f1cb23d7-vmware-logo.png",
-                            alt="VMware",
-                            width="257",
-                            height="256",
-                            hi_def=True,
-                            loading="lazy",
-                            attrs={"class": "p-logo-section__logo"},) | safe
-              }}
-            </div>
-            <div class="p-logo-section__item">
-              {{ image(url="https://assets.ubuntu.com/v1/9faa9b91-aws-logo.png",
-                            alt="AWS",
-                            width="125",
-                            height="256",
-                            hi_def=True,
-                            loading="lazy",
-                            attrs={"class": "p-logo-section__logo"},) | safe
-              }}
-            </div>
-          </div>
-        </div>
+      <p>Get comprehensive, 24/7 or weekday support for Charmed Kafka, covering the entire solution for up to 10 years.</p>
+      <div class="p-cta-block">
+        <a class="p-button" href="">Learn more</a>
```

Comment:
CTA is still wip and will 404 for now, see discussion [here](https://docs.google.com/document/d/1bfo97XdFdKlzrMcMoRxiTdzt_bPVndx6j55IPsapuAE/edit?disco=AAABLcLNUYI)